### PR TITLE
add copy button to docs

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

**Review by SteveFlames — COMMENTED** [review]
We should update the changelog as well.
Other than that looks good :+1:

**Comment by irmakyz** [comment]
> We should update the changelog as well. Other than that looks good 👍

thanks! 

- [ ] add the changelog

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@geoblink/design-system",
-  "version": "35.4.0",
+  "version": "35.5.0-beta.1",
```
Comment (fofi) [inline package.json:3]: Why moving to to 35.5?

Comment (irmakyz) [reply package.json:3]: because is a minor feature i thought not a bug

**Review by dletran — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Navas23) [reply package.json:3]: I think this is a patch, we are not adding functionality to the design system, we are just improving the docs

Comment (irmakyz) [reply package.json:3]: hmmm, oki then i will change it

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@geoblink/design-system",
-  "version": "35.4.0",
+  "version": "35.4.2-beta.0",
```
Comment (Navas23) [inline package.json:3]: Why 35.4.2 and not 35.4.1?

Comment (dletran) [reply package.json:3]: @irmakyz you still have this question pending

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 35.4.2
```
Comment (SteveFlames) [inline CHANGELOG.md:3]: Make sure you update the version here as well

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 35.4.1
+New:
+- Copy code button is added
```
Comment (Navas23) [inline CHANGELOG.md:5]: Sorry for not saying this earlier, but we need to be more specific here. Mention we added a button to copy code in the docs

**Review by Navas23 — APPROVED** [review]

**Review by fofi — APPROVED** [review]

**Review by SteveFlames — APPROVED** [review]

**Pull request merged**